service/dap/server.go

```diff
@@ -1900,13 +1900,13 @@ func (s *Server) convertVariableWithOpts(v *proc.Variable, qualifiedNameOrExpr s
 	// Those that are fully missing (e.g. due to hitting MaxVariableRecurse), can be reloaded in place.
 	// Those that are partially missing (e.g. MaxArrayValues from a large array), need a more creative solution
 	// that is still to be determined. For now, clearly communicate when that happens with additional value labels.
-	// TODO(polina): look into layered/paged loading for truncated strings, arrays, maps and structs.
```

Comment:
Also don't need comments on the previous likes as well "need a more creative solution" for large arrays, etc.